### ISSUE TITLE
NameError: name 'LightManager' is not defined

### ISSUE THREAD

**Issue by Jraylward0** [GammaGames/kano-wand-demos#4]
When running hue.py using the tutorial posted on medium, i came across the following error:
```
_Traceback (most recent call last):
  File "hue.py", line 60, in <module>
    class LightManager():
  File "hue.py", line 123, in LightManager
    manager = LightManager()
NameError: name 'LightManager' is not defined_
```

I tried several different work arounds but nothing seemed to work?

**Comment by GammaGames** [comment]
Have you added the `LightManager` class to the script? You can find the completed version [here](https://github.com/GammaGames/kano-wand-demos/blob/master/hue.py#L59).

**Comment by Jraylward0** [comment]
Sorry about that, silly mistake on my part. I was trying to replicate the error I was having the other day with the multiple calls for _light.state_

`Traceback (most recent call last):
  File "hue1.py", line 154, in <module>
    main()
  File "hue1.py", line 123, in main
    manager = LightManager()
  File "hue1.py", line 96, in __init__
    light.state(on=True, **self.color_values[None])
TypeError: Resource object got multiple values for keyword argument 'on'`

**Comment by Jraylward0** [comment]
We just need to edit out line 96, the duplicate states is creating an error. Thanks for everything! This has been really fun to play around with.

**Comment by GammaGames** [comment]
Oh, I see how it's happening! I'm not sure why I wasn't having the issue before, but it is on [line 95](https://github.com/GammaGames/kano-wand-demos/blob/master/hue.py#L95) where it's passing the `on` keyword and then using the keywords from `self.color_values[None]`

**Comment by newsroomdev** [comment]
I’m also seeing the same error about multiple values for the keyword “on” while running this on a Raspberry Pi 4 @GammaGames Should I comment out the aforementioned line or try a different Linux distro?

**Comment by GammaGames** [comment]
Sorry, should be fixed in ff7694e05bf18c8bdcb76b2c3dce2a82101e59bd! If I don't hear back in a week or so I'll close the issue.
